doc/maintaining/configuration.rst

```diff
@@ -270,6 +270,23 @@ Default value:  ``True``
 This option allows you to disable the datastore_search_sql action function, and
 corresponding API endpoint if you do not wish it to be activated.
 
+.. _ckan.datastore.resource_query.enabled:
+
+ckan.datastore.resource_query.enabled
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Example::
+
+ ckan.datastore.resource_query.enabled = False
+
+Default value:  ``True``
```

Comment:
I'm not sure how I feel about having this feature enabled by default. It is definitely an advance feature and I think most users will be confused by it. My vote is for defaulting it to False.

Reply:
I'm fine with that.